cli/tsc/mod.rs

```diff
@@ -402,7 +402,10 @@ impl State {
 }
 
 fn normalize_specifier(specifier: &str) -> Result<ModuleSpecifier, AnyError> {
-  resolve_url_or_path_deprecated(specifier).map_err(|err| err.into())
+  // TODO(bartlomieju): ideally we shouldn't need to call `current_dir()` on each
+  // call - maybe it should be caller's responsibility to pass it as an arg?
+  let cwd = std::env::current_dir().context("Unable to get CWD")?;
```

Comment:
I think we should cache this in the state.

Reply:
Yeah I think so too, but didn't want to do more changes to the places I wasn't sure about.